offchain/inspect-server/src/server.rs

```diff
@@ -31,11 +32,11 @@ pub fn create(
             .wrap(middleware::Logger::default())
             .wrap(cors)
             .service(
-                web::resource(inspect_get_path.clone())
+                web::resource(INSPECT_GET_PATH)
```

Comment:
You can add `#[get("/inspect/{payload:.*}")]` to the function, like this https://github.com/cartesi/rollups-node/blob/main/offchain/host-runner/src/http/rollup_server.rs#L40

Reply:
My bad. Done at fa9b8da38f806c364ec793cff771ff7ee396873c